timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -228,27 +228,30 @@ types:
       LeaderTimes:
         fields:
           leaderTimes: map<Namespace, LeaderTime>
-      TimestampLeaseRequest:
-         fields:
-           requestId:
-             type: uuid
-             safety: safe
-           numFreshTimestamps:
-             type: integer
-             safety: safe
+      RequestId:
+        alias: uuid
+        safety: safe
```

Comment:
I can throw this away if needed. do we really need two safe-uuid wrappers

Reply:
Do we have another one?

Reply:
More wrappers always better

Reply:
We have the one for `LeaseIdentifier` but I guess the name is what we have these for